addons/repair/tests/test_repair.py

```diff
@@ -417,34 +415,10 @@ def test_02_repair_sale_order_binding(self):
         for line in repair_order.move_ids:
             line.quantity = line.product_uom_qty
         repair_order.action_repair_end()
-        self.assertTrue(float_is_zero(order_line.qty_delivered, 2))
+        self.assertEqual(float_compare(order_line.product_uom_qty, order_line.qty_delivered, 2), 0)
```

Comment:
```suggestion
        self.assertEqual(order_line.product_uom_qty, order_line.qty_delivered)
```
In this case this should be enough. In case you really need to handle float precision, you can use `assertAlmostEqual` instead.